File: 7-kyu/the-barksdale-code/javascript/solution.js
const decode = string => {
  const lookupTable = {"0": "5", "1": "9", "2": "8", "3": "7", "4": "6", "5": "0", "6": "4", "7": "3", "8": "2", "9": "1"};
  return string.split('').map(elem => lookupTable[elem]).join('');
}

File: 7-kyu/the-barksdale-code/python/solution.py
def decode(strng):
    lookup_table = {'0': '5', '1': '9', '2': '8', '3': '7', '4': '6', '5': '0', '6': '4', '7': '3', '8': '2', '9': '1'}
    return ''.join(lookup_table[n] for n in strng)